src/modules/mavlink/mavlink_messages.cpp

```diff
@@ -2004,42 +2066,55 @@ class MavlinkStreamLocalPositionNEDCOV : public MavlinkStream
 
 protected:
 	explicit MavlinkStreamLocalPositionNEDCOV(Mavlink *mavlink) : MavlinkStream(mavlink),
+		_pos_sub(_mavlink->add_orb_subscription(ORB_ID(vehicle_local_position))),
+		_pos_time(0),
 		_est_sub(_mavlink->add_orb_subscription(ORB_ID(estimator_status))),
 		_est_time(0)
 	{}
 
 	bool send(const hrt_abstime t)
 	{
-		struct estimator_status_s est = {};
+		vehicle_local_position_s pos = {};
```

Comment:
Leads to overhead - better zero it only later if needed.

Reply:
sure.